detection-rules/abuse_docsend_new_domain.yml

```
@@ -0,0 +1,37 @@
+name: "Service Abuse: DocSend Share From New Domain"
```

```suggestion
name: "Service Abuse: DocSend Share From Newly Registered Domain"
```
Thoughts on increasing verbosity here? the "new" in "new domain" make me think of "new to my org"